src/main/java/org/jbake/parser/MarkupEngine.java

```diff
@@ -140,6 +141,31 @@ public void processBody(final ParserContext context) {}
             content.put(Crawler.Attributes.TAGS, tags);
         }
         
+    	// If categories are not disabled then add a default category
+		if(config.getBoolean(Keys.CATEGORIES_ENABLE)){
+			if (content.get("categories") != null) {
+	        	String[] categories = (String[]) content.get("categories");
+	            for( int i=0; i<categories.length; i++ ) {
+	            	categories[i]=categories[i].trim();
+	                if (config.getBoolean(Keys.CATEGORY_SANITIZE)) {
+	                	categories[i]=categories[i].replace(" ", "-");
+	                }
+	            }
+	            content.put("categories", categories);
+	        } else {
+        		String defaultCategory = config.getString(Keys.CATEGORY_DEFAULT).trim();
+        		if (config.getBoolean(Keys.CATEGORY_SANITIZE,false)) {
+        			defaultCategory = defaultCategory.replace(" ", "-");
+                }
+        		content.put("categories", ArrayUtils.toArray(defaultCategory));
```

Comment:
Also here it would be better to use Crawler.Attributes.CATEGORIES instead of the string value "categories"